### ISSUE TITLE
Proxyman and Openfortivpn

### ISSUE THREAD

**Issue by dsilva85** [ProxymanApp/Proxyman#1842]
Hi guys!

Im very new here, so please be patient with me :)

Whenever i use openfortivpn, i dont get any activity on my Proxyman. Is there anything i can do?

Thanks a bunch!

**Comment by NghiaTranUIT** [comment]
Hey, can you check this docs: https://docs.proxyman.io/troubleshooting/proxyman-does-not-work-with-vpn-apps

and google if `openfortivpn` supports Proxy Config, then it can work with Proxyman.

**Comment by dsilva85** [comment]
Yeah, i gave it a go, but i honestly have no clue. Sorry for being such a newbie, but would really appreciate some help with this.
